js/style/style_function.js

```diff
@@ -8,7 +8,9 @@ exports.interpolated = function(parameters) {
         return inner(globalProperties && globalProperties.zoom, featureProperties || {});
     };
     outer.isFeatureConstant = inner.isFeatureConstant;
-    outer.isGlobalConstant = inner.isGlobalConstant;
+    outer.isZoomConstant = inner.isZoomConstant;
+    outer.getStopZoomLevels = inner.getStopZoomLevels;
```

Comment:
What led to your design decision to create a `getStopZoomLevels` method on `mapbox-gl-function` rather than parse these out within `mapbox-gl-js`?